UNLV-CS472-672/2024-S-GROUP5-Munch
# Updated Spacing and placement for create page

This is related to issue #

## Description / Changes Made

- fixed spacing and positioning for create page to align with branding

## Portions of this code that utilized AI generation

- Link to Generated Code (If Applicable): n/aa

## Screenshots (optional)

![image](#)

![Untitled](https://github.com/UNLV-CS472-672/2024-S-GROUP5-Munch/assets/136287986/58faeced-cc0f-4864-97e8-c39149e0502d)

## How to Test

1. open create
2. ensure it is still on the page

## Checklist
